p2p/messageMonitor.go

```diff
@@ -15,19 +17,36 @@ const (
 	PublishReq      = "0" //PublishReq
```

Comment:
can u use enum instead of hard code ?

Reply:
more places to change, the next version will optimized